src/test/java/seedu/ccacommander/logic/commands/UnenrolCommandTest.java

```diff
@@ -0,0 +1,197 @@
+        Index eventIndex = Index.fromZeroBased(1);
+        UnenrolCommand unenrolCommand = new UnenrolCommand(outOfBoundMemberIndex, eventIndex);
+
+        assertCommandFailure(unenrolCommand, model, Messages.MESSAGE_INVALID_MEMBER_DISPLAYED_INDEX);
+    }
+
+    @Test
+    public void execute_invalidEventIndexUnfilteredList_throwsCommandException() {
+        Index memberIndex = Index.fromZeroBased(1);
+        Index outOfBoundEventIndex = Index.fromZeroBased(model.getFilteredEventList().size() + 1);
+        UnenrolCommand unenrolCommand = new UnenrolCommand(memberIndex, outOfBoundEventIndex);
+
+        assertCommandFailure(unenrolCommand, model, Messages.MESSAGE_INVALID_EVENT_DISPLAYED_INDEX);
+    }
+
+    @Test
+    public void execute_invalidMemberEventIndexesUnfilteredList_throwsCommandException() {
+        Index outOfBoundMemberIndex = Index.fromZeroBased(model.getFilteredMemberList().size() + 1);
+        Index outOfBoundEventIndex = Index.fromZeroBased(model.getFilteredEventList().size() + 1);
+        UnenrolCommand unenrolCommand = new UnenrolCommand(outOfBoundMemberIndex, outOfBoundEventIndex);
+
+        assertCommandFailure(unenrolCommand, model, Messages.MESSAGE_INVALID_MEMBER_AND_EVENT_DISPLAYED_INDEX);
+    }
+
+    @Test
+    public void equals() {
+        UnenrolCommand unenrolFirstCommand = new UnenrolCommand(INDEX_FIRST_MEMBER, INDEX_FIRST_EVENT);
+        UnenrolCommand unenrolSecondCommand = new UnenrolCommand(INDEX_SECOND_MEMBER, INDEX_SECOND_EVENT);
+
+        // same object -> returns true
+        assertTrue(unenrolFirstCommand.equals(unenrolFirstCommand));
+
+        // same values -> returns true
+        UnenrolCommand unenrolFirstCommandCopy = new UnenrolCommand(INDEX_FIRST_MEMBER, INDEX_FIRST_EVENT);
+        assertTrue(unenrolFirstCommand.equals(unenrolFirstCommandCopy));
+
+        // different types -> returns false
+        assertFalse(unenrolFirstCommand.equals(1));
+
+        // null -> returns false
+        assertFalse(unenrolFirstCommand.equals(null));
+
+        // different member -> returns false
+        assertFalse(unenrolFirstCommand.equals(unenrolSecondCommand));
+    }
+
+    @Test
+    public void toStringMethod() {
+        Index memberIndex = Index.fromOneBased(1);
+        Index eventIndex = Index.fromOneBased(1);
+        UnenrolCommand unenrolCommand = new UnenrolCommand(memberIndex, eventIndex);
+        String expected = UnenrolCommand.class.getCanonicalName() + "{memberIndex=" + memberIndex
+                + ", eventIndex=" + eventIndex + "}";
+        assertEquals(expected, unenrolCommand.toString());
+    }
+
+    /**
+     * Updates {@code model}'s filtered enrolment list to show no one.
+     */
+    @Test
+    private void showNoEnrolment(Model model) {
+        model.updateFilteredEnrolmentList(p -> false);
+
+        assertTrue(model.getFilteredEnrolmentList().isEmpty());
+    }
+}
```

Comment:
I would recommend to remove this as it seems to be testing more for model instead of unenrolCommand unless it's used.